misc/releases/templates/_Vagrantfile.tpl

```diff
@@ -37,75 +41,28 @@ Vagrant.configure("2") do |config|
         wget            \
```

Comment:
I think `wget` can also go here (like above for soplex).

Reply:
Thanks. I noticed it only after merging but fixed it in a [trivial] commit.